docs/UserGuide.md

```diff
@@ -124,20 +124,19 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Deleting a client : `delClient`
 
-Deletes the specified person from the address book.
+Deletes the specified client from MyInsuRec.
 
-Format: `delete INDEX`
+Format: `delClient i/INDEX`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+* Deletes the client at the specified `INDEX`.
+* The index refers to the index number shown in the displayed clients' list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
-
+* `list` followed by `delClient 2` deletes the 2nd client in the address book.
```

Comment:
Please follow the example format written by other people
change it to `delClient i/2`

Reply:
okay :+1: